src/components/Entry.vue

```diff
@@ -1,13 +1,15 @@
 <template>
-  <div></div>
+
 </template>
 
 <script>
 export default {
+  props: ['Name', 'Address', 'Telephone', 'Email'],
```

Comment:
And for here, we'll probably have to use more generic props.
Cause these props are very specific to the PersonalDetails component.
Maybe we'll also make use of slots to make sure that our components are more versatile